service/src/main/java/greencity/service/UserServiceImpl.java

```diff
@@ -927,6 +927,14 @@ public void markUserAsDeactivated(String uuid) {
         userRepo.save(user);
     }
 
+    @Override
+    public void markUserAsActivated(String uuid) {
+        User user = userRepo.findUserByUuid(uuid).orElseThrow(
+            () -> new NotFoundException(ErrorMessage.USER_NOT_FOUND_BY_UUID));
+        user.setUserStatus(UserStatus.ACTIVATED);
```

Comment:
should we check if the user status is `inactive` before setting it to `active`?

Reply:
no, it will checks from ui and its not important for this method